src/main/java/io/cdap/plugin/salesforce/plugin/sink/batch/SalesforceSinkConfig.java

```diff
@@ -133,6 +134,12 @@ public class SalesforceSinkConfig extends ReferencePluginConfig {
   @Description("The existing connection to use.")
   private SalesforceConnectorConfig connection;
 
+  @Name(PROPERTY_DATATYPE_VALIDATION)
+  @Nullable
+  @Macro
```

Comment:
do you need a containsMacro() check for this field?

Reply:
will add that.